src/app/visual/summary-actionbar/summary-actionbar-modal-demo.component.ts

```diff
@@ -8,7 +8,8 @@ import {
 
 @Component({
   selector: 'sky-test-cmp-modal',
-  templateUrl: './summary-actionbar-modal-demo.component.html'
+  templateUrl: './summary-actionbar-modal-demo.component.html',
+  styleUrls: ['./summary-actionbar-demo.component.scss']
```

Comment:
Small, but the name of the SCSS file should be `./summary-actionbar-modal-demo.component.scss`, to be consistent with the template.

Reply:
I just hadn't made a separate file for the modal compared to the main part of the demo. Probably best to do that for future proofing. Done :)